tracing-mock/src/subscriber.rs

```diff
@@ -56,10 +245,58 @@ impl MockSubscriberBuilder {
     /// (*technically*, the name of the thread where it was created, which is
     /// the name of the test unless tests are run with `--test-threads=1`).
     /// When a test has only one mock subscriber, this is sufficient. However,
-    /// some tests may include multiple subscribers, in order to test
-    /// interactions between multiple subscribers. In that case, it can be
+    /// some tests may include multiple subscriber, in order to test
+    /// interactions between multiple subscriber. In that case, it can be
     /// helpful to give each subscriber a separate name to distinguish where the
     /// debugging output comes from.
+    ///
+    /// # Examples
+    ///
+    /// In the following example, we create 2 subscribers, both
```

Comment:
in general, numbers are spelled out when single digit:
```suggestion
    /// In the following example, we create two subscribers, both
```